src/decays/specializations/Ah/decay_Ah_to_dbard.inc

```diff
@@ -132,13 +132,29 @@ double CLASSNAME::get_partial_width<PseudoscalarHiggs, bar<DownTypeQuark>::type,
    }
 
    // low x limit
-   double result_DR =
+   const double result_DR =
       flux * color_factor * phase_spaceDR * amp2DR_P;
    // high x limit
-   double result_OS =
+   const double result_OS =
       flux * color_factor * phase_spaceOS * amp2OS_P;
 
    const double result = (1-4.*xOS)*result_DR + 4*xOS*result_OS;
 
+   if (static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_higgstools)) ||
```

Comment:
See my comment on `static_cast<bool>`

Reply:
Done in c587517522b7b65208aa28eb3ac3b70a03a3ce92.